build-system/git.js

```diff
@@ -24,55 +24,45 @@ const {getStdout} = require('./exec');
 
 const commitLogMaxCount = 100;
 
-/**
- * Returns the merge base of the current branch off of master when running on
- * a local workspace.
- * @return {string}
- */
-exports.gitMergeBaseLocalMaster = function() {
-  return getStdout('git merge-base master HEAD').trim();
-};
-
 /**
  * Returns the merge base of the current branch off of master, regardless of
  * the running environment.
  * @return {string}
  */
 exports.gitMergeBaseMaster = function() {
   if (process.env.TRAVIS) {
-    const commitRange = process.env.TRAVIS_COMMIT_RANGE.split('...');
-    return getStdout(`git merge-base ${commitRange[0]} ${commitRange[1]}`)
-        .trim();
+    return getStdout(
+        `git merge-base master ${process.env.TRAVIS_PULL_REQUEST_SHA}`).trim();
```

Comment:
Curious: Why is this `master` and not `origin/master`? Does it even make a difference? (If not, no worries.)

Reply:
Edit: Since you've run a lot of tests with the code as it is, ignore this.

Reply:
For local development, some users will have this called `upstream/master`, so I'd rather not include the `remote` name